apps/jira-node/src/services/monday-service.js

```diff
@@ -0,0 +1,108 @@
+const fetch = require('node-fetch');
+const initMondayClient = require('monday-sdk-js');
+
+const triggerMondayIntegration = async (webhookUrl, data = {}) => {
+  fetch(webhookUrl, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      Authorization: process.env.MONDAY_SIGNING_SECRET,
+    },
+    body: JSON.stringify({
+      trigger: {
+        outputFields: {
+          ...data,
+        },
+      },
+    }),
+  });
+};
+
+const queryColumns = async (token, boardId) => {
+  try {
+    const mondayClient = initMondayClient({ token });
+    const query = `query queryBoards($boardId: [Int]) { boards(ids: $boardId) { columns { id } } }`;
+    const variables = { boardId };
+    var response = await mondayClient.api(query, { variables });
```

Comment:
better not to use var, but let (or on this case just const) and after

return response?.data[0]?.columns